init/systemd/ipa.sysusers.base.conf

```diff
@@ -0,0 +1,5 @@
+u kdcproxy  -   "IPA KDC Proxy User"    -
+u dirsrv    -   -   -
```

Comment:
@tiran You're right, I omitted the GECOS because create_ds_user didn't set it either.
@HonzaCholasta Doesn't matter if the user is or isn't created by some other package. That's the benefit of sysusers.d, we declare that we need such user and sysusers.d creates it doesn't exist already.